site/DataPageContent.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect } from "react"
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons/faArrowDown"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import { Grapher, GrapherInterface } from "@ourworldindata/grapher"
+import { ExpandableAnimatedToggle } from "./ExpandableAnimatedToggle.js"
+import ReactDOM from "react-dom"
+import { GrapherWithFallback } from "./GrapherWithFallback.js"
+
+declare global {
+    interface Window {
+        _OWID_DATAPAGE_PROPS: any
+    }
+}
+
+export const OWID_DATAPAGE_CONTENT_ROOT_ID = "owid-datapage-root"
+
+export const DataPageContent = ({
+    datapage,
+    grapherConfig,
+}: {
+    datapage: any
+    grapherConfig: GrapherInterface
+}) => {
+    const [grapher, setGrapher] = React.useState<Grapher | undefined>(undefined)
+
+    // Initialize the grapher for client-side rendering
+    useEffect(() => {
+        setGrapher(new Grapher(grapherConfig))
+    }, [grapherConfig])
+
+    return (
+        <>
+            <div className="DataPageContent__grapher-for-embed">
+                <GrapherWithFallback
+                    grapher={grapher}
+                    slug={grapherConfig.slug}
+                />
+            </div>
+            <div className="DataPageContent">
+                <div className="header__wrapper wrapper">
+                    <div className="header__left">
+                        <div className="supertitle">DATA</div>
```

Comment:
nit: To make things nicer for people who may copy and paste these sections, I prefer writing these with normal title case and using `text-transform: uppercase;`

Reply:
Also I recognize this is WIP, but I just want to note for the final version that I'd prefer these classes were written with BEM. `.title` is generic and could end up accidentally getting affected by other components throughout the site.

Reply:
yes, I'm relying on `.title` not being defined at the top level indeed. I was roughly doing BEM down the tree until I felt the extra verbosity wasn't justified. But maybe simpler to just go all the way all the time.

> To make things nicer for people who may copy and paste these sections

@ikesau Not sure about the workflow you're referring to here, and how "DATA" vs "Data" affects it. Turns out, `overline-black-caps` already does the uppercase tranform anyway, but curious still.